torchvision/models/detection/_utils.py

```diff
@@ -140,6 +140,10 @@ def __init__(self, weights, bbox_xform_clip=math.log(1000. / 16)):
         self.weights = weights
         self.bbox_xform_clip = bbox_xform_clip
 
+    @torch.jit.export
+    def __getstate__(self):
+        return self.weights, self.bbox_xform_clip
```

Comment:
why was this added ? usually __getstate__ is accompanied by a corresponding __setstate__

Reply:
Thanks.
For listing the model parameters associated with attributes in user defines classes, we are relying on getstate method:
https://github.com/pytorch/pytorch/blob/7a0be0fcc6b03d8014e96eb3d58b54b385a1b6c0/torch/csrc/jit/passes/onnx/list_model_parameters.cpp#L132

This helps listing the parameters for RCNN models when scripting.

Reply:
We should try to find another way to get parameters than relying on getstate, there should be other alternatives

Reply:
Using getstate is only for user defined classes, and it's been used in lower_graph pass similarly.
Do you have any suggestions for handling custom classes?

Reply:
can you get all the attributes listed in the class type ?